plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -217,6 +217,19 @@ Job create(JobInfo jobInfo)
         return bigQuery.create(jobInfo);
     }
 
+    public TableResult query(String sql)
+    {
+        log.debug("Execute query: %s", sql);
+        try {
+            // Don't call another query method because setting create disposition in jobs with DDL statements is unsupported
```

Comment:
Should this comment be on the caller side?

Also should we rename the method to clarify it's purpose? Something like `executeDDL` or `ddlQuery`?

Reply:
I changed to `void executeUpdate` likes JDBC driver. Also, removed the comment it looks redundant now.